src/main/java/seedu/apollo/ui/Ui.java

```diff
@@ -329,7 +330,7 @@ public void printSpecificTimetable(Module module, LessonType lessonType, ArrayLi
         for (Timetable timetable : copyList) {
             System.out.println("Class Number: " + timetable.getClassNumber());
             System.out.println("   " + timetable.getDay() + " " + timetable.getStartTime() + " - " +
-                    timetable.getEndTime());
+                                timetable.getEndTime() + " " + timetable.compressedWeeks(timetable));
```

Comment:
I think the indentation for this line is off, please check!

Reply:
It's right on IntelliJ